cpu/cortexm_common/ldscripts/cortexm.ld

```diff
@@ -23,12 +23,16 @@
 _rom_offset = DEFINED( _rom_offset ) ? _rom_offset : 0x0;
 _fw_rom_length = DEFINED( _fw_rom_length ) ? _fw_rom_length : _rom_length - _rom_offset;
 
+_backup_ram_start_addr = DEFINED( _backup_ram_start_addr ) ? _backup_ram_start_addr : 0x0 ;
+_backup_ram_len = DEFINED( _backup_ram_len ) ? _backup_ram_len : 0x0 ;
+
 ASSERT((_fw_rom_length <= _rom_length - _rom_offset), "Specified firmware size does not fit in ROM");
 
 MEMORY
 {
     rom (rx)    : ORIGIN = _rom_start_addr + _rom_offset, LENGTH = _fw_rom_length
     ram (w!rx)  : ORIGIN = _ram_start_addr,               LENGTH = _ram_length
+    bkup-ram    : ORIGIN = _backup_ram_start_addr,        LENGTH = _backup_ram_len
```

Comment:
```suggestion
    bkup-ram (w!rx) : ORIGIN = _backup_ram_start_addr,    LENGTH = _backup_ram_len
```

Also, I think this new approach is better than having a separate ldscript. With a default length of 0, any attemt to use ".backup" where it is not supported will cause a link failure.

Reply:
fixed.

Reply:
The `(w!rx)` is of course just a nitpick.